idaes/models/costing/SSLW.py

```diff
@@ -238,14 +238,20 @@ def build_process_costs(self):
     @staticmethod
     def initialize_build(self):
         """
-        Here we can add intialization steps for the things we built in
+        Here we can add initialization steps for the things we built in
         build_process_costs.
 
         Note that the aggregate costs will be initialized by the framework.
         """
         # TODO: For now,  no additional process level costs to initialize
         pass
 
+    def cost_none(blk, integer=True):
```

Comment:
Is this method actually required? When we wrote the new API, we explicitly allowed for users to provide their own methods during costing - i.e. when registering costing for a unit model, the method used does not need to come from the same library as the costing block, and users can (and should) provide their own methods where applicable.

Thus, this sounds like a case where the users should be doing the following:

```
model.fs.unit.costing = UnitModelCostingBlock(
        flowsheet_costing_block=model.fs.costing,
        costing_method=my_costing_method,
        costing_method_arguments={**my_args},
    )
```

Reply:
@andrewlee94 The method is not required to use the costing framework. However, as currently implemented the model must build a group of unnecessary variables and constraints in order to define the `UnitModelCostingBlock`, such as in the code snippet in the PR summary. Adding a method that does nothing allows users to add a total plant cost constraint without fully defining the SSLW equipment cost arguments or having to write their own "empty" costing method.

Reply:
Using the example above, it would look something like:

```
def my_costing_method(blk, *args):
    @blk.Expression()
    def total_plant_cost(blk):
        extra_installed_area = 1.10  # accounts for cell degradation
        return pyunits.convert(60.87 * blk.unit_model.soec_module.number_cells *
                               extra_installed_area * pyunits.USD_2018,
                               CE_index_units)
```

Reply:
@andrewlee94 Would `my_costing_method` need to be attached to a `FlowsheetCostingBlock` class, or could it just exist as a regular function?

Reply:
It can be a stand-alone function, i.e. you can declare it in the flowsheet module immediately before using it.